ia/src/protocole.py

```
@@ -0,0 +1,128 @@
+import socket
+import threading
```

Ruff (0.11.9)

`threading` imported but unused

Remove unused import: `threading`

(F401)